modules/observe/scorecards/proc-configuring-jira-scorecards-in-rhdh-instance.adoc

```diff
@@ -0,0 +1,177 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-configuring-jira-scorecards-in-rhdh-instance_{context}"]
+= Configuring Jira Scorecards to view GitHub metrics in your {product} instance
```

Comment:
is the title correct? Do we use JIRA scorecards to view GitHub metrics or project metrics??